bin/server.py

```diff
@@ -74,6 +74,8 @@ def __init__(self, application, request, **kwargs):
         self._base_packages_query = {}
         self._selected_packages = []
 
+        self._configuration = dict(defaults.CONFIGURATION)
```

Comment:
* Add `time_zone` item to `defaults.CONFIGURATION`;
* Rename `hostname` to `host_name`, `SSID` to `WPA_SSID` and `PSK` to `WPA_PSK`;